### ISSUE TITLE
HammerJS ToolTip issues

### ISSUE THREAD

**Issue by ToonvanStrijp** [angular/material2#5758]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
HammerJS adding styles to tooltip:
`style="touch-action: none; user-select: none; -webkit-user-drag: none; -webkit-tap-highlight-color: rgba(0, 0, 0, 0);"`

#### What is the current behavior?
The elements where I add mdTooltip="test", don't get the HammerJS Styling

#### What are the steps to reproduce?
Can't reproduce it in plunkr since im not using SystemJS

**Comment by donroyco** [comment]
Please follow the issue template.

**Comment by ToonvanStrijp** [comment]
@donroyco I discovered the bug. It's a really strange bug in material2. When I  put `MdSliderModule` in imports then all the elements that have mdTooltip are getting the HammerJs Styling: `style="touch-action: none; user-select: none; -webkit-user-drag: none; -webkit-tap-highlight-color: rgba(0, 0, 0, 0);"`. Can you maybe give me an explanation why this is necessary to work?

**Comment by andrewseguin** [comment]
Can you re-open with a plunker showing a reproduction of what the issue is?

**Comment by ToonvanStrijp** [comment]
Without hammerJS styling
http://plnkr.co/edit/IUb3IZVDMiDhcnVhQKZA?p=preview

With hammerJS styling
http://plnkr.co/edit/6Nhbz64S1pHG6c6Vvkjo?p=preview

Above two plunkers with an example, the first plunker is without the hammerJS styling, inside the main.ts file I removed the MatSliderModule. (When this module is included it causes to get HammerJS activated sort of... )

The second plunker I added the MatSliderModule again and if you go to inspect element en target the button It will have the hammerJS styling.

Hopefully you understand my problem now? It's not blocking at al or anything but I was just surprised why it works this way.
